packages/client/tests/functional/relationMode-17255-same-actions/_matrix.ts

```diff
@@ -0,0 +1,62 @@
+import { defineMatrix } from '../_utils/defineMatrix'
+import { computeMatrix } from '../_utils/relationMode/computeMatrix'
+
+// const RelationModeEnv = process.env.RELATION_MODE
+// if (RelationModeEnv && RelationModeEnv !== 'prisma' && RelationModeEnv !== 'foreignKeys') {
+//   throw new Error(`RELATION_MODE must be either "prisma" or "foreignKeys" but was "${RelationModeEnv}"`)
+// }
+
+// type RelationMode = 'prisma' | 'foreignKeys' | ''
+// const relationMode: RelationMode = (RelationModeEnv as RelationMode) || ''
+
+// const defaultMatrix = computeMatrix({ relationMode })
+// console.log(defaultMatrix)
+
+// SetNull (everywhere) fails with:
+// error: Error parsing attribute "@relation": The `onDelete` referential action of a relation must not be set to `SetNull` when a referenced field is required.
+// Either choose another referential action, or make the referenced fields optional.
+// So we filter these out
+//
```

Comment:
Do we need this block of comments?

Reply:
The last part about SetNull is good to keep

The part up is only good if someone wants to develop on these tests maybe.

I'll keep them for now.